File: 프로그래머스/1/86051. 없는 숫자 더하기/없는 숫자 더하기.cs
using System;

public class Solution {
    public int solution(int[] numbers) {
        int answer = 45; //1부터 9까지 더한 수
        
        for(int i = 0; i < numbers.Length; i++) {
            answer -= numbers[i];
        }
        return answer;
    }
}

File: 프로그래머스/1/86051. 없는 숫자 더하기/없는 숫자 더하기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> numbers) {
    int answer = 45; //1부터 9까지 더한 수
    
    for(int i = 0; i < numbers.size(); i++) {
        answer -= numbers[i];
    }
    return answer;
}